testing/unit_tests/test_loader.py

```diff
@@ -216,5 +216,51 @@ def test_2d_patches(download_data_testing_test_files,
     assert len(ds) == 16
 
 
+@pytest.mark.parametrize('loader_parameters', [{
+    "path_data": [os.path.join(__data_testing_dir__, "microscopy_png")],
+    "bids_config": f"{path_repo_root}/ivadomed/config/config_bids.json",
+    "target_suffix": ["_seg-myelin-manual"],
```

Comment:
maybe we should test with a list of 2 target suffix, ex: with ["_seg_myelin_manual", "_seg-axon-manual"]. I tried it but it does not work, I'm not sure but I think `target_filename` may be overwritten in the for loop.